content/docs/communities/moderation/_index.md

```diff
@@ -95,21 +93,280 @@ in the message bar at the bottom:
+
+To do so, you can issue a variant of the command used to ban an individual user.
+The command was
+
+```
+!mjolnir ban <list shortcode> <user|room|server> <glob> [reason]
+```
+
+As you probably guessed, you are not going to ban a user but a server this once.
+For example, to ban the `maliciousdomain.tld` domain:
+
+```
+!mjolnir ban spam server maliciousdomain.tld
+```
+
+Mjolnir is going to add the domain to the `spam` ban list, and whenever someone
+from that homeserver tries to join your community they are going to be banned.
+
+### Creating ban lists
+
+Mjolnir allows you to create several ban lists that it can later watch. Those
+ban lists can also be watched by third parties if you make them public.
+
+The command to issue in mjolnir's control room to create a list is the
+following
+
+```
+!mjolnir list create <shortcode> <alias localpart>
+```
+
+- `shortcode` is a short name given to this list. It should be short and easy to
+  type, since you are likely to type it a lot when banning users.
+- `alias localpart` is the local part of the address mjolnir is going to create
+  for this list. This is useful if you ever want to [share your ban list with
+  other communities](#publishing-ban-lists).
+
+For example, assuming the server mjolnir is deployed against is `example.com`,
+the following command will create a ban list with the short code `spam` and the
+address `#my-community-spam-ban-list:example.com`
+
+```
+!mjolnir list create spam my-community-spam-ban-list
+```
+
+You only need to create a ban list once, and can then add as many users and
+servers as you want to that ban list. You can also create as many ban lists as
+you want.
+
+Communities often create two ban lists: one for spam, and one for code of
+conduct infringements.
+
+### Subscribing to ban lists
+
+Ban lists are a clever mechanism that allows moderation teams to ban users for
+different motives (e.g. one list for `spam` and one for `coc`). Such a
+distinction can be useful when several communities want to collaborate together.
+
+Not all communities will share a similar Code of Conduct, but a lot of them will
+agree on what is spam. Being able to subscribe to another community's spam list
+means your own community will be protected from spammers the other community has
+already met, all while observing different code of conducts.
+
+To subscribe to a public ban list, you need to retrieve the address of this
+list. That list being technically nothing more than a Matrix room, its address
+follows the usual `#room_name:server.tld` format.
+
+Then to make mjolnir follow this list, you need to issue the following command
+in its control room
+
+```
+!mjolnir watch <room alias/ID>
+```
+
+For example to subscribe to the `#matrix-org-coc-bl:matrix.org` ban list
+maintained by the Matrix Foundation, you would issue the following command
+
+```
+!mjolnir watch #matrix-org-coc-bl:matrix.org
+```
+
+### Publishing ban lists
```

Comment:
This probably can moved into the creation of the lists. It feels out of place here and isn't long enough to warrant a full section imho

Reply:
Fixed in [bec4c28](https://github.com/matrix-org/matrix.org/pull/1617/commits/bec4c28a9d72b1459bbe492e141a844e8873fd6d)[bec4c28](https://github.com/matrix-org/matrix.org/pull/1617/commits/bec4c28a9d72b1459bbe492e141a844e8873fd6d)